stable/filebeat/README.md

```diff
@@ -56,7 +56,9 @@ The following table lists the configurable parameters of the filebeat chart and
 | `affinity`                                               |                                                                                                          | `{}`                                               |
 | `rbac.create`                                            | Specifies whether RBAC resources should be created                                                       | `true`                                             |
 | `serviceAccount.create`                                  | Specifies whether a ServiceAccount should be created                                                     | `true`                                             |
-| `serviceAccount.name`                                    | he name of the ServiceAccount to use                                                                     | `""`                                               |
+| `serviceAccount.name`                                    | the name of the ServiceAccount to use                                                                     | `""`                                               |
```

Comment:
`podSecurityPolicy.enabled` not documented.  looks like the `podSecurityPolicy.name` is misdocumented with `enabled` description ?

Reply:
Thanks for the catch, `name` is not needed as it uses the fullname from the helpers. This has been fixed in https://github.com/helm/charts/pull/9151/commits/d73a3a217feecce864a6c593d652f56c0b21cf39